client/lib/purchases/index.js

```diff
@@ -163,6 +167,15 @@ function isRenewing( purchase ) {
 	return includes( [ 'active', 'autoRenewing' ], purchase.expiryStatus );
 }
 
+function isSubscription( purchase ) {
```

Comment:
It would probably help to add a docblock and explain the concept of subscription. We use this term quite a lot everywhere and here it has a specific meaning, mostly for presentation.